### PR TITLE
Fix missing object type

### PR DESCRIPTION
**Pull request by sy-records**
Is this missing?  2761e65ea261c33f65a7854b567a5821cf06aaa7

### CODE REVIEW

File: reference/classobj/functions/is-a.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </methodsynopsis>
   <para>
    Checks if the given <parameter>object_or_class</parameter> is of this object type or has
-   this class as one of its parents.
+   this object type as one of its parents.
```
Comment (cmb69) [inline reference/classobj/functions/is-a.xml:18]: I think when we speak of types, referring to a "parent" is uncommon; instead we probably want to call it "supertype".

Note that we should fix the return-value section as well. And actually, some other manpages, too, e.g. `is_subclass_of()`, although in that case the [name of the function is misleading](https://3v4l.org/dupND). :(

**Comment by mumumu** [comment]
If we accept this PR, should also fix return type description. See also #1796.

**Comment by sy-records** [comment]
Thank you! @mumumu

File: reference/classobj/functions/is-a.xml (hunk 1 above)
Comment (sy-records) [reply reference/classobj/functions/is-a.xml:18]: Do I need to change it to `supertype`?

**Review by Elimuhub-coder — COMMENTED** [review]
change location to Nairobi/Kenya[location](https://maps.google.com/maps/contrib/117273694586709139832) 
@sy-records

File: reference/classobj/functions/is-a.xml (hunk 1 above)
Comment (afilina) [reply reference/classobj/functions/is-a.xml:18]: @cmb69 Could you suggest a change that you had in mind so that it's clearer?

Comment (cmb69) [inline reference/classobj/functions/is-a.xml:18]: I'd change
````diff
- this object type as one of its parents.
+ this object type as one of its supertypes.
````

File: reference/classobj/functions/is-a.xml [hunk 2]
```diff
@@ -53,7 +53,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   Returns &true; if the object is of this class or has this class as one of
+   Returns &true; if the object is of this object type or has this object type as one of
    its parents, &false; otherwise.
```
Comment (cmb69) [inline reference/classobj/functions/is-a.xml:57]:
````diff
- its parents, &false; otherwise.
+ its supertypes, &false; otherwise.
````

**Comment by sy-records** [comment]
Resubmitted a PR #2238

**Pull request closed without merging**